forem/forem
# Remove mascot footer image

## What type of PR is this? (check all applicable)


## Description

This PR removes all settings related to the mascot footer image. Note that after removing these settings there were only 2 left in `Settings::Mascot`. so it no longer makes sense to have a dedicated model for that so I moved the remaining settings back to `SiteConfig`. Note that we can only remove the `settings_mascots` table after this PR has been deployed across the fleet.

## Related Tickets & Documents

#13587

## QA Instructions, Screenshots, Recordings

Go to http://localhost:3000/admin/customization/config and ensure the Mascot section works as expected.

### UI accessibility concerns?

None

## Added tests?

- [X] Yes, updated existing ones
 
## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._


## [optional] Are there any post deployment tasks we need to perform?

- [ ] Ping @citizen428 after this got deployed across the fleet so we can remove the settings table
